packages/vector_graphics_compiler/lib/vector_graphics_compiler.dart

```diff
@@ -60,6 +60,7 @@ Future<Uint8List> encodeSvg(String input, String filename) async {
             ? Float32List.fromList(shader.offsets!)
             : null,
         tileMode: shader.tileMode.index,
+        transform: shader.transform?.toMatrix4(),
```

Comment:
We should avoid passing this in if `shader.transform == AffineMatrix.identity`, otherwise we're wasting space and CPU cycles on identity information.